ms2rescore/__init__.py

```diff
@@ -245,4 +249,34 @@ def run(self):
         if self.config["general"]["run_percolator"]:
             self._run_percolator()
 
+        logger.info("Generating Rescore plots")
+        if self.config["general"]["plotting"]:
+            plotting.PIN(
+                self.config["general"]["output_filename"]
+                + "_"
+                + "_".join(self.config["general"]["feature_sets"][0])
+                + "_features.pin",
```

Comment:
Code clarity would improve if this string would be constructed beforehand. Idem below for pout